tests/testthat/test-fetch.R

```diff
@@ -40,11 +41,15 @@ test_that('fetch', {
 
   paramsFetchNow[c('run', 'keep')] = NULL
 
-  result = do.call(fetch, c(
+  resultObs = do.call(fetch, c(
     list(remoteFilepaths = metadataGSM[[remoteColname]], outputDir = outputDirFetchTest),
     paramsFetchNow))
 
-  for (file in strsplit(result$localFilepaths, ';')[[1]]) {
+  resultExp = snapshot(resultObs, file.path(dataDir, 'fetch_output_testing.qs'))
+
+  expect_equal(resultObs, resultExp)
+
+  for (file in strsplit(resultExp$localFilepaths, ';')[[1]]) {
     file = paste0('./', file)
```

Comment:
Why is this line necessary?

Reply:
My previous question was regarding paste0.

Reply:
Gotcha, this is now resolved